src/main/resources/processes/SMC.bpmn

```diff
@@ -50,6 +50,7 @@
         <camunda:out source="CompType" target="CompType" />
         <camunda:out source="Stage" target="Stage" />
         <camunda:out source="ScmTriageResult" target="ScmTriageResult" />
+        <camunda:out source="CctTriageAccept" target="CctTriageAccept" />
```

Comment:
Is the SCM triage step different to the CCT triage step? I might need to see a demo of this to properly understand this flow